packs/data/spell-effects.db/spell-effect-wild-morph.json

```
@@ -60,6 +64,270 @@
+                                "not": "wild-morph:long-vines"
+                            }
+                        ],
+                        "value": "long-vines"
+                    },
+                    {
+                        "img": "systems/pf2e/icons/spells/angelic-wings.webp",
+                        "label": "PF2E.SpecificRule.WildMorph.Wings",
+                        "predicate": [
+                            "feat:soaring-shape",
+                            {
+                                "not": "wild-morph:wings"
+                            }
+                        ],
+                        "value": "wings"
+                    }
+                ],
+                "flag": "spellEffectWildMorph2",
+                "key": "ChoiceSet",
+                "predicate": [
+                    {
+                        "gte": [
+                            "item:level",
+                            6
+                        ]
+                    },
+                    {
+                        "or": [
+                            {
+                                "and": [
+                                    "feat:wild-shape",
+                                    {
+                                        "not": "wild-morph:wild-claws"
+                                    }
+                                ]
+                            },
+                            {
+                                "and": [
+                                    "feat:insect-shape",
+                                    {
+                                        "not": "wild-morph:wild-jaws"
+                                    }
+                                ]
+                            },
+                            {
+                                "and": [
+                                    "feat:elemental-shape",
+                                    {
+                                        "not": "wild-morph:elemental-matter"
+                                    }
+                                ]
+                            },
+                            {
+                                "and": [
+                                    "feat:plant-shape",
+                                    {
+                                        "not": "wild-morph:long-vines"
+                                    }
+                                ]
+                            },
+                            {
+                            
+                                "and": [
+                                    "feat:soaring-shape",
+                                    {
+                                        "not": "wild-morph:wings"
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ],
+                "prompt": "PF2E.SpecificRule.WildMorph.Prompt2",
+                "rollOption": "wild-morph2"
```

```suggestion
                "rollOption": "wild-morph"
```
Shouldn't need to differentiate the roll options, and making them the same simplifies the predicates.